scripts/core/registry.yml

```diff
@@ -376,9 +376,18 @@ etors:
 - name: COMMAND_BUFFER_FINALIZE_EXP
   desc: Enumerator for $xCommandBufferFinalizeExp
   value: '123'
+- name: BINDLESS_IMAGES_UNSAMPLED_IMAGE_HANDLE_DESTROY_EXP
+  desc: Enumerator for $xBindlessImagesUnsampledImageHandleDestroyExp
+  value: '124'
```

Comment:
After you rebase - I would recommend reverting this file back to it's state in main and rerunning the generator. we have made changes to the way the registry values are allocated and we will no longer back-fill gaps.

Reply:
Done.